packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -1179,7 +1185,7 @@ class ThemeData with Diagnosticable {
   ///   * Typography: `typography` (see table above)
   ///
   /// ### Components
-  ///   * Common buttons: [TextButton], [OutlinedButton], [ElevatedButton]
+  ///   * Common buttons:  [ElevatedButton], [FilledButton], [OutlinedButton], [TextButton]
```

Comment:
Whitespace typo (?) 👀

```suggestion
  ///   * Common buttons: [ElevatedButton], [FilledButton], [OutlinedButton], [TextButton]
```

Reply:
Ooops. Fixed. Thx.